applications/ContactStructuralMechanicsApplication/custom_conditions/ALM_frictional_mortar_contact_axisym_condition.cpp

```diff
@@ -31,7 +31,7 @@ Condition::Pointer AugmentedLagrangianMethodFrictionalMortarContactAxisymConditi
     NodesArrayType const& rThisNodes,
     PropertiesPointerType pProperties ) const
 {
-    return boost::make_shared< AugmentedLagrangianMethodFrictionalMortarContactAxisymCondition<TNumNodes, TNormalVariation> >( NewId, this->GetGeometry().Create( rThisNodes ), pProperties );
+    return Condition::Pointer( new  AugmentedLagrangianMethodFrictionalMortarContactAxisymCondition<TNumNodes, TNormalVariation >( NewId, this->GetGeometry().Create( rThisNodes ), pProperties ));
```

Comment:
why did u change from make_shared?

Reply:
to remove the boost:: dependency

Reply:
make_shared has a higher performance at creation and destruction time.

you are creating/destructing this condition many times...so here it could make a difference